controller/konnect/ops/ops_errors.go

```diff
@@ -202,14 +202,20 @@ func SDKErrorIsConflict(sdkError *sdkkonnecterrs.SDKError) bool {
 		return false
 	}
 
-	if sdkErrorBody.Message != dataConstraintMesasge {
-		return false
-	}
+	const (
+		dataConstraintMesasge      = "data constraint error"
+		typeUniqueConstraintFailed = "(type: unique) constraint failed"
+	)
 
-	switch sdkErrorBody.Code {
-	case 3, 6:
-		return true
+	if sdkErrorBody.Message == dataConstraintMesasge ||
+		strings.Contains(sdkErrorBody.Message, typeUniqueConstraintFailed) {
+
+		switch sdkErrorBody.Code {
+		case 3, 6:
```

Comment:
Could we add some comments (better if we can provide some references) about the error codes? `case 3, 6` looks too magic.

Reply:
Good point, added more docs and exemplary body in 28eaa105fd7150a446e8c8d4d35f11c092df1bae

PTAL